### PR TITLE
add AWS onboarding information for GovCloud

### PR DESCRIPTION
**Pull request by afeld**
Fixes https://github.com/18F/cg-product/issues/53.

Took a pass at it. I'm sure there are things missing, but at least it's a start.

/cc @NoahKunin @18F/cloud-gov-ops

### CODE REVIEW

File: content/ops/aws-onboarding.md [hunk 1]
```diff
@@ -5,10 +5,38 @@ menu:
...
+The procedures for working with Amazon Web Services (AWS) differ between the two environments, and thus members of Atlas are given different levels of access.
+
+### US-East
+
 If you are going to be a Cloud Foundry operator you need to be onboarded to AWS.
 
 Things you will need:
 
 - Access to EC2, RDS and CloudFormation
 - Your key added to the jumpbox
```
Comment (mogul) [inline content/ops/aws-onboarding.md]: The jumpbox no longer exists, in either environment.

File: content/ops/aws-onboarding.md [hunk 2]
```diff
@@ -5,10 +5,38 @@ menu:
...
+
+<!-- needed for compliance -->
+
+#### Levels of access
+
+While we try to minimize interaction with AWS directly (explained in next section), there are three different levels of AWS IAM access, provided to different roles:
+
+* **Full administrative accounts:** This is be provided to the people known in our compliance documentation as the "Cloud Operations team".
+* **Read-only accounts:** This is provided to GSA Security staff, who need access for incident response.
+* **Indirect access through jumpboxes:** While the remainder of the Atlas team isn't given an AWS IAM account directly, they are able to SSH in to access the various BOSH machines by creating jumpboxes. See the [Troubleshooting BOSH](../troubleshooting-bosh/) page for more information.
```
Comment (mogul) [inline content/ops/aws-onboarding.md]: s/jumpboxes/ephemeral jumpboxes/ (just to make the point that they're not setting up their own personal jumpboxes)

**Review by mogul — COMMENTED** [review]
A good start, though I'd want someone with more specific knowledge than me to sign-off. Thanks!

File: content/ops/aws-onboarding.md (hunk 1 above)
Comment (afeld) [reply content/ops/aws-onboarding.md]: Really? Not in us-east?

Comment (afeld) [reply content/ops/aws-onboarding.md]: ...or do we now use the ephemeral ones in both environments?

Comment (mogul) [reply content/ops/aws-onboarding.md]: I say again...

> The jumpbox no longer exists, in either environment.

File: content/ops/metrics.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ menu:
 title: Metrics
 ---
 
+***This page is out of date.***
```
Comment (afeld) [inline content/ops/metrics.md:8]: Not really sure _how much_ of this page is no longer accurate, would help for someone that knows more than me to go through it.

https://github.com/18F/cg-docs/issues/447

**Pull request merged**